cmd/geth/main.go

```diff
@@ -326,6 +324,9 @@ func geth(ctx *cli.Context) error {
 		return fmt.Errorf("invalid command: %q", args[0])
 	}
 
+	// Automatically set GOMAXPROCS to match Linux container CPU quota.
+	maxprocs.Set(maxprocs.Logger(func(string, ...interface{}) {}))
```

Comment:
you can just `maxprocs.Set()`, it won't log by default according to https://github.com/uber-go/automaxprocs/issues/18#issuecomment-511330567

Reply:
thanks for the advice